### PR TITLE
ISSD-1330 Set up "Browse by Capability" on the homepage 

### PR DESCRIPTION
**Pull request by abfranca**

### CODE REVIEW

**Review by nNilton — APPROVED** [review]

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-capability-cards/ddm-template.ftl [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+							${taxonomyCategory.description}
+						</p>
+					</div>
+				</a>
+			</div>
+		</#list>
+	</div>
+</#if>
+
+<style>
```
Comment (nNilton) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-capability-cards/ddm-template.ftl:41]: move the style to the start of the article

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by abfranca** [comment]
ci:test:sf

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-capability-cards/ddm-template.ftl (hunk 1 above)
Comment (abfranca) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-capability-cards/ddm-template.ftl]: Done.

**Review by nNilton — APPROVED** [review]

**Comment by joseabelenda** [comment]
resent:https://github.com/brianchandotcom/liferay-portal/pull/145215

**Pull request closed without merging**